app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -306,6 +306,52 @@ class ProfileChooserFragmentTest {
     }
   }
 
+  @Test
+  fun testProfileChooserFragment_clickAddProfileWithNoPin_OpensAdminPinActivity_checkExtras() {
```

Comment:
This test can be merged with `testProfileChooserFragment_clickAdminProfileWithNoPin_checkOpensAdminPinActivity`
all you need to add the extra check at the end 
`intended(hasExtra(ADMIN_PIN_ENUM_EXTRA_KEY, AdminAuthEnum.PROFILE_ADD_PROFILE.value))`

Same with others.

Reply:
So instead of 4 extra tests, I have to add a line in the existing tests right

Reply:
Yes

Reply:
Done